packages/kbn-ci-stats-performance-metrics/BUILD.bazel

```diff
@@ -40,9 +40,8 @@ RUNTIME_DEPS = [
   "//packages/kbn-dev-cli-runner",
   "//packages/kbn-test",
   "//packages/kbn-tooling-log",
-  "@npm//@elastic/elasticsearch",
-  "@npm//axios-curlirize",
-  "@npm//qs",
+  "//packages/kbn-ci-stats-reporter",
+  "@npm//lodash",
```

Comment:
I don't see any lodash usage (and would push against it :P) so I think you can drop this and the @types dep